Alignment/OfflineValidation/plugins/TrackerOfflineValidation.cc

```diff
@@ -417,7 +417,7 @@ TrackerOfflineValidation::checkBookHists(const edm::EventSetup& es)
 {
   es.get<TrackerDigiGeometryRecord>().get( tkGeom_ );
   const TrackerGeometry *newBareTkGeomPtr = &(*tkGeom_);
-  if (newBareTkGeomPtr == bareTkGeomPtr_) return; // already booked hists, nothing changed
+  if (newBareTkGeomPtr == bareTkGeomPtr_) { return; // already booked hists, nothing changed }
```

Comment:
I've tried to reproduce this wrong substitution with a minimal test case, and I haven't been able to.
What options did you use ?

Reply:
@davidlange6 and @fwyzard this was due to cmssw build rules. We run clang-tidy with `-export-fixes` option and then post process the generated yaml files to 
- drop suggested change for files which are not part of PR change set
- remove duplicate changes for headers files

Problem is that when we write back the yaml file (using pyYAML) then it drops the extra new line e.g. if originally we have
```
Diagnostics:     
  - DiagnosticName:  readability-braces-around-statements
    Message:         statement should be inside braces
    FileOffset:      566
    FilePath:        /build/muz/cf/CMSSW_10_6_X_2019-04-11-2300/src/Alignment/CocoaModel/interface/MeasurementCOPS.h
    Replacements:    
      - FilePath:        /build/muz/cf/CMSSW_10_6_X_2019-04-11-2300/src/Alignment/CocoaModel/interface/MeasurementCOPS.h
        Offset:          566
        Length:          0
        ReplacementText: ' {'
      - FilePath:        /build/muz/cf/CMSSW_10_6_X_2019-04-11-2300/src/Alignment/CocoaModel/interface/MeasurementCOPS.h
        Offset:          591
        Length:          0
        ReplacementText: '
}'
```

then reading it and writing back generates
```
Diagnostics:     
  - DiagnosticName:  readability-braces-around-statements
    Message:         statement should be inside braces
    FileOffset:      566
    FilePath:        /build/muz/cf/CMSSW_10_6_X_2019-04-11-2300/src/Alignment/CocoaModel/interface/MeasurementCOPS.h
    Replacements:    
      - FilePath:        /build/muz/cf/CMSSW_10_6_X_2019-04-11-2300/src/Alignment/CocoaModel/interface/MeasurementCOPS.h
        Offset:          566
        Length:          0
        ReplacementText: ' {'
      - FilePath:        /build/muz/cf/CMSSW_10_6_X_2019-04-11-2300/src/Alignment/CocoaModel/interface/MeasurementCOPS.h
        Offset:          591
        Length:          0
        ReplacementText: ' }'
```

this causes the closing braces to be on the same line. This should be fixes once we have https://github.com/cms-sw/cmsdist/pull/4914 merged.